src/main.c

```diff
@@ -761,6 +761,33 @@ static NETX_CONSOLEAPP_RESULT_T opMode_spiMacroPlayer(tFlasherInputParameter *pt
 }
 
 
+/* ------------------------------------- */
+
+static NETX_CONSOLEAPP_RESULT_T opMode_identify(tFlasherInputParameter *ptAppParams, NETX_CONSOLEAPP_PARAMETER_T *ptConsoleParams)
```

Comment:
This could be done in a different way, as there is already a routine to show a repeating blink pattern. Like this (not tested, see blinki): 
static NETX_CONSOLEAPP_RESULT_T opMode_identify()
{
	BLINKI_HANDLE_T tBlinkiHandle;
	unsigned long ulStartTimeMs;
	const unsigned long ulBlinkDurationMs = 5000U;

	/* Switch all LEDs off. */
	rdy_run_setLEDs(RDYRUN_OFF);

	ulStartTimeMs = systime_get_ms();
	rdy_run_blinki_init(&tBlinkiHandle, 0x00000055, 0x00000150);
	while (0 == systime_elapsed(ulStartTimeMs, uBlinkDurationMs))
	{
		rdy_run_blinki(&tBlinkiHandle);
	};

	return NETX_CONSOLEAPP_RESULT_OK;
}